src/mjolnir/valhalla_add_elevation.cc

```diff
@@ -0,0 +1,121 @@
+#include <cstdlib>
+#include <iostream>
+#include <vector>
+
+#include <boost/program_options.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "baldr/graphtile.h"
+#include "baldr/rapidjson_utils.h"
+#include "config.h"
+#include "mjolnir/elevationbuilder.h"
+
+namespace bpo = boost::program_options;
+
+using namespace valhalla::baldr;
+using namespace valhalla::midgard;
+using namespace valhalla::mjolnir;
+
+enum Input { CONFIG, TILES };
+
+/*
+ * This service downloads elevations from remote storage for each provided tile.
```

Comment:
ok this isnt a service though right, its a program that runs once and terminates. its not a daemon.